samples/tcp_server.cr

```diff
@@ -20,4 +20,7 @@ end
 
 server = TCPServer.new "127.0.0.1", 9000
 puts "listen on 127.0.0.1:9000"
-loop { spawn process(server.accept) }
+loop do
+  client = server.accept
+  spawn { process(client) }
```

Comment:
Notice here you are spawning with a closured variable, so you could end up processing the same client twice and loose others if the `server.accept` is executed again before the `process` method starts.

As you can see, you committed the same mistake we did on the beginning, so seems like this approach is not totally intuitive after all :smiley: